doc/guides/using-internal-errors.md

```diff
@@ -15,8 +15,8 @@ to be breaking changes. By providing a permanent identifier for a specific
 error, we reduce the need for userland code to inspect error messages.
 
 *Note*: Switching an existing error to use the `internal/errors` module must be
-considered a `semver-major` change. However, once using `internal/errors`,
-changes to `internal/errors` error messages will be handled as `semver-minor`
+considered a `semver-major` change. In the future, after the transition
+has been completed, error message changes should be handled as `semver-minor`
```

Comment:
Makes sense.
Originally I thought we should end with a "hopeful" note. But that's sort of the subject of the guide as a whole.